assets/constants.js

```diff
@@ -474,6 +474,8 @@ var constants = {
 
   "USE_NEW_SUGGESTION_FRAMEWORK": false,
 
+  "ENABLE_GENERALIZED_FEEDBACK_THREADS": false,
```

Comment:
Either have this in constants.js or feconf.py, but not both. It is possible to import values from constants.js in the backend code.